app/services/record_declaration.rb

```diff
@@ -13,7 +13,7 @@ class RecordDeclaration
   attribute :evidence_held
   attribute :has_passed
 
-  before_validation :validate_if_npq_course_supported
+  before_validation :ensure_npq_course_no_longer_supported
```

Comment:
We can just remove this method i think

Reply:
hmm, I'm thinking this is there to prevent any npq declarations from being sent accidentally. ignore my comment